7Cav-Alive.Altis/scripts/missionEnd/missionEnd_mech.sqf

```diff
@@ -0,0 +1,58 @@
+private _boxNumber = missionNameSpace getVariable "endMissionNumber";

```

Comment:
Don't repeat code. Take one of the mission end files, structure it as a function with some parameters, and just call it four times instead of four identical files